kuri_camera/src/uds_to_tcp.cpp

```diff
@@ -16,6 +16,14 @@ int main(int argc, char **arcv) {
     return -1;
   }
 
+  // Sleep for some time to ensure that the masumx daemon has fully started
+  int sleep_secs;
+  if (!ros::param::get(ros::this_node::getName()+"/sleep_secs", sleep_secs))
+  {
+    sleep_secs = 1;
+  }
+  ros::Duration(sleep_secs).sleep();
```

Comment:
It would be _preferable_ to use some sign-of-life. Maybe checking to see if the stream files have been created in `/var/run/madmux/` would do it.

Reply:
Used error checking to resolve this.